lib/system.nim

```diff
@@ -149,6 +149,18 @@ else:
   template runnableExamples*(body: untyped) =
     discard
 
+when defined(nimHasAlias):
+  proc aliasImpl[T1, T2](name: T1, expr: T2) {.magic: "Alias".}
+
+  template `:=`*(name, expr) =
```

Comment:
I don't think a new operator should be introduced at this point in `system.nim`. Every piece of Nim code depends on this module, and if they have their own `:=`, it would likely conflict.

Reply:
done, moved to sugar.nim